server/lib/chat/chat-api.ts

```diff
@@ -190,20 +198,38 @@ export class ChatApi {
     return await this.chatService.getChannelUsers(channelName, ctx.session!.userId)
   }
 
-  @httpDelete('/:channelName/:targetId')
+  @httpGet('/:channelName/:targetId')
```

Comment:
You should be *really* careful when you add routes like this, as they can collide with previous routes. Yes, using user IDs here minimizes risk, but it's still a risk and I don't think we should be taking it. On top of that, this just isn't really very RESTful. This should be `/:channelName/users/:targetId` (as should all other routes that deal with specific users)